src/components/common/PeopleAndRoles/ListPeopleAndRoles.vue

```diff
@@ -553,6 +553,11 @@ export default class ListPeopleAndRoles extends Mixins(CommonMixin) {
   }
 }
 
+// adjust error bar left spacing
+.people-roles-content.section-container.invalid-section {
+  margin-left: -30px
```

Comment:
Yes, this is the right CSS property, can you just fix line 467 instead?

Reply:
If I change line 467, then the left and the left and right margins won't be the same. This happens only when the error bar shows up, with the "invalid-section" class.

Reply:
Ah, OK...

Then let's use the power of SCSS to nest selectors: somewhere in the `.people-roles-content {` block, add this sub-block:
```
  // adjust ...
  &invalid-section {
    margin-left: -30px;
  }
```

Reply:
Looks good.